evt.js

```diff
@@ -64,6 +65,16 @@
     }
   }
 
+  // If there is an error listner, delegate to that, otherwise just log the
+  // errors, so that emit notifications to other listeners still work.
+  function emitError(err) {
+    if (evt._events.hasOwnProperty('error')) {
+      evt.emit('error', err);
+    } else {
+      console.error(err + '\n' + err.stack);
```

Comment:
To play nicer with devtools that may do special formatting for err.stack, perhaps then I'll just do `console.error(err, err.stack)`. I am concerned that the '\n' + part will result in coercing to a string. I'll try that out in gaia to make sure it works out there with our overrides, and then merge in that form if it looks OK. 

I liked the idea of keeping it on a separate line from the message, but that was a minor formatting pref when in string form, feel like the possibility of better devtools display is more valuable.